docs/schema/lobby.md

```diff
@@ -847,9 +852,112 @@ export interface LobbyLeftEventData {
 ```
 ---
 
+## ListReset
+
+Sent by the server to give the client the full list of all lobbies
+
+- Endpoint Type: **Event**
+- Source: **Server**
+- Target: **User**
+- Required Scopes: `tachyon.lobby`
+
+### Event
+
+<details>
+<summary>JSONSchema</summary>
+
+```json
+{
+    "title": "LobbyListResetEvent",
+    "tachyon": {
+        "source": "server",
+        "target": "user",
+        "scopes": ["tachyon.lobby"]
+    },
+    "type": "object",
+    "properties": {
+        "type": { "const": "event" },
+        "messageId": { "type": "string" },
+        "commandId": { "const": "lobby/listReset" },
+        "data": {
+            "title": "LobbyListResetEventData",
+            "type": "object",
+            "properties": {
+                "lobbies": {
+                    "type": "object",
+                    "patternProperties": {
+                        "^(.*)$": { "$ref": "#/definitions/lobbyOverview" }
+                    }
+                }
+            },
+            "required": ["lobbies"]
+        }
+    },
+    "required": ["type", "messageId", "commandId", "data"]
+}
+
+```
+</details>
+
+<details>
+<summary>Example</summary>
+
+```json
+{
+    "type": "event",
+    "messageId": "sint",
+    "commandId": "lobby/listReset",
+    "data": {
+        "lobbies": {
+            "JH+": {
+                "id": "nulla eiusmod",
+                "name": "ad ullamco sunt ipsum proident",
+                "playerCount": 26791585,
+                "maxPlayerCount": -87375796,
+                "mapName": "amet proident non elit",
+                "engineVersion": "in nostrud",
+                "gameVersion": "ullamco esse cillum enim",
+                "currentBattle": null
+            }
+        }
+    }
+}
+```
+</details>
+
+#### TypeScript Definition
+```ts
+export type UnixTime = number;
+
+export interface LobbyListResetEvent {
+    type: "event";
+    messageId: string;
+    commandId: "lobby/listReset";
+    data: LobbyListResetEventData;
+}
+export interface LobbyListResetEventData {
+    lobbies: {
+        [k: string]: LobbyOverview;
+    };
+}
+export interface LobbyOverview {
+    id: string;
+    name: string;
+    playerCount: number;
+    maxPlayerCount: number;
+    mapName: string;
+    engineVersion: string;
+    gameVersion: string;
+    currentBattle: {
```

Comment:
Shouldn't ``currentBattle`` be optional here too?

Reply:
Yes good point, I amended the commit.